assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -90,7 +90,82 @@ describe("DiversionPage", () => {
     expect(screen.queryByTitle("Detour End")).not.toBeInTheDocument()
   })
 
-  test("directions start with origin intersection when second waypoint is added", async () => {
+  test("has a normal cursor before placing the start point", async () => {
+    const { container } = render(<DiversionPage />)
+
+    await waitFor(() => {
+      expect(
+        container.querySelector(".c-detour_map--map__clickable")
+      ).not.toBeInTheDocument()
+    })
+  })
+
+  test("has a waypoint cursor, given by the __clickable class, once the start point is placed", async () => {
+    const { container } = render(<DiversionPage />)
+
+    act(() => {
+      fireEvent.click(originalRouteShape.get(container))
+    })
+
+    await waitFor(() => {
+      expect(
+        container.querySelector(".c-detour_map--map__clickable")
+      ).toBeInTheDocument()
+    })
+  })
+
+  test("has no waypoints at the beginning or when placing the start point", async () => {
+    const { container } = render(<DiversionPage />)
+
+    expect(
+      container.querySelectorAll(".c-detour_map-circle-marker--detour-point")
+    ).toHaveLength(0)
```

Comment:
I think it would be valuable to test that (and thus it shall be done), but I don't think that behavior is governed by the existence of `finishDetour`, so I'm not sure this is coverage we lost (although I agree that it's coverage we should have)

I think the existence of `finishDetour` governs whether the `"Finish Detour"` button appears, which is tested (implicitly) [here](https://github.com/mbta/skate/pull/2580/files#diff-1535a72a38a9500ad346d1b3d9735aff2c788a2d97cfbf49f2aeaac7aefb26cfR261-R262)

Reply:
[Feedback: Add test that explicitly validates the initial state](https://github.com/mbta/skate/pull/2580/commits/d296898b1aa0d6c4a0ecb3854d066e1d8ad01be9)

It shall be done, and thusly, it has been done!

(It also enabled me to remove an assertion from a different test, which was also contorting itself to test this, since this explicit coverage was missing!)